src/EditorFeatures/CSharpTest/SplitComment/SplitCommentCommandHandlerTests.cs

```diff
@@ -0,0 +1,394 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Editor.UnitTests.SplitComment;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.SplitComment
+{
+    [UseExportProvider]
+    public class SplitCommentCommandHandlerTests : AbstractSplitCommentCommandHandlerTests
+    {
+        protected override TestWorkspace CreateWorkspace(string markup)
+            => TestWorkspace.CreateCSharp(markup);
+
+        [WorkItem(38516, "https://github.com/dotnet/roslyn/issues/38516")]
+        [WpfFact, Trait(Traits.Feature, Traits.Features.SplitComment)]
+        public void TestWithSelection()
+        {
+            TestHandled(
+@"public class Program
+{
+    public static void Main(string[] args) 
+    { 
+        //[|Test|] Comment
+    }
+}",
+@"public class Program
+{
+    public static void Main(string[] args) 
+    { 
+        //
+        //Comment
```

Comment:
Shouldn't there be a space between slashes and `Comment`?
```suggestion
        // Comment
```

Reply:
I took the policy that left point of the selection helped indicate how much space to consume.